### PR TITLE
Incorrect total in response meta

### LINKED ISSUE #1563: Incorrect total in response meta
### Actual behavior

```jsonc
{
  "data": [
    ...               // only two eligible entries
  ],
  "meta": {
    "count": 2,
    "offset": 0,
    "total": 5        // <--- actual
  }
}
```

### Expected behavior

```jsonc
{
  "data": [
    ...               // only two eligible entries
  ],
  "meta": {
    "count": 2,
    "offset": 0,
    "total": 2        // <--- expected
  }
}
```

### Steps to reproduce

- Invoke the `/blockchain/apps/meta?network=devnet` endpoint
- Verify the same for the `/blockchain/apps/meta/tokens` and `/blockchain/apps/meta/list` endpoints

### Which version(s) does this affect? (Environment, OS, etc...)

Lisk Service v0.7.0-alpha.11

### PR DESCRIPTION
**Pull request by vardan10**
### What was the problem?
This PR resolves #1563

### How was it solved?

- [x] Updated query to fetch total from db for `/blockchain/apps/meta` endpoint
- [x] Updated query to fetch total from db for `/blockchain/apps/meta/tokens` endpoint
- [x] Updated query to fetch total from db for `/blockchain/apps/meta/list` endpoint

### How was it tested?

- [x] local

### CODE REVIEW

**Review by priojeetpriyom — APPROVED** [review]

File: services/blockchain-app-registry/shared/metadata.js [hunk 1]
```diff
@@ -212,7 +212,7 @@ const getBlockchainAppsMetadata = async (params) => {
 		{ concurrency: blockchainAppsMetadata.data.length },
 	);
 
-	const [{ total }] = await applicationMetadataTable.rawQuery(`SELECT COUNT(chainName) as total from ${applicationMetadataIndexSchema.tableName}`);
+	const total = await applicationMetadataTable.count(params, ['chainName']);
```
Comment (nagdahimanshu) [inline services/blockchain-app-registry/shared/metadata.js:215]: Can you please verify if `count` method accepts second params?

Comment (vardan10) [reply services/blockchain-app-registry/shared/metadata.js:215]: Yes, you are right. It accepts only a single parameter. Will update.

**Review by nagdahimanshu — APPROVED** [review]
LGTM and works as expected 👍

**Comment by nagdahimanshu** [comment]
@vardan10 Can we also please verify the same for `/blockchain/apps/meta/tokens` endpoint. I think the `total` is incorrect there as well.

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Review by priojeetpriyom — APPROVED** [review]

**Pull request merged**